### PR TITLE
[image_picker] Updated docs getImage() about preference rear front not working on Android

### PR DESCRIPTION
**Pull request by valconetti**
## Description

Updated the documentation of getImage() in image_picker.dart and image_picker_platform.dart about Android's disability to preference front/rear camera, because this wasn't specified and caused confusion for people wondering why it didn't work.

## Related Issues

[Issue 55644](https://github.com/flutter/flutter/issues/55644)

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [X] All existing and new tests are passing.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [X] I read and followed the [Flutter Style Guide].
- [X] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [X] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [X] I updated CHANGELOG.md to add a description of the change.
- [X] I signed the [CLA].
- [X] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [X] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by mvanbeusekom** [comment]
@googlebot I fixed it.

File: packages/image_picker/image_picker/pubspec.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: image_picker
 description: Flutter plugin for selecting images from the Android and iOS image
   library, and taking new pictures with the camera.
 homepage: https://github.com/flutter/plugins/tree/master/packages/image_picker/image_picker
-version: 0.6.7+7
+version: 0.6.8
```
Comment (cyanglaz) [inline packages/image_picker/image_picker/pubspec.yaml:5]: Since it is just a documentation update, we should only update the patch version, so 0.6.7+8
https://dart.dev/tools/pub/versioning#semantic-versions

File: packages/image_picker/image_picker/lib/image_picker.dart [hunk 2]
```diff
@@ -87,7 +87,8 @@ class ImagePicker {
   ///
   /// Use `preferredCameraDevice` to specify the camera to use when the `source` is [ImageSource.camera].
   /// The `preferredCameraDevice` is ignored when `source` is [ImageSource.gallery]. It is also ignored if the chosen camera is not supported on the device.
-  /// Defaults to [CameraDevice.rear].
+  /// Defaults to [CameraDevice.rear]. Since Android has no documented parameter for an intent to specify if the front or rear camera should be opened, this
```
Comment (cyanglaz) [inline packages/image_picker/image_picker/lib/image_picker.dart:90]: Maybe the `Since Android..` part should be a new line. Ditto below.

**Review by cyanglaz — COMMENTED** [review]
LGTM % the versioning and some nits on the documentation.

File: packages/image_picker/image_picker/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 0.6.8
+
+* Update documentation of getImage() about Android's disability to preference front/rear camera
```
Comment (cyanglaz) [inline packages/image_picker/image_picker/CHANGELOG.md:3]: nits: 
```suggestion
* Update documentation of getImage() about Android's disability to preference front/rear camera.
```

**Review by cyanglaz — COMMENTED** [review]
We need to bump the version for the platform_interface plugin as well

File: packages/image_picker/image_picker/pubspec.yaml (hunk 1 above)
Comment (mvanbeusekom) [reply packages/image_picker/image_picker/pubspec.yaml]: Changed version number to 0.6.7+8 (missed the part which explained the semantic versioning prior version 1.0.0).

File: packages/image_picker/image_picker/lib/image_picker.dart (hunk 2 above)
Comment (mvanbeusekom) [reply packages/image_picker/image_picker/lib/image_picker.dart]: Done, I updated the documentation to (also in the image_picker_platform_interface):

```
  /// Defaults to [CameraDevice.rear]. 
  /// 
  /// Note that Android has no documented parameter for an intent to specify if
  /// the front or rear camera should be opened, this function is not guaranteed
  /// to work on an Android device.
```

File: packages/image_picker/image_picker/CHANGELOG.md (hunk 3 above)
Comment (mvanbeusekom) [reply packages/image_picker/image_picker/CHANGELOG.md]: Added the extra period.

File: packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart [hunk 4]
```diff
@@ -132,7 +132,11 @@ abstract class ImagePickerPlatform extends PlatformInterface {
   ///
   /// Use `preferredCameraDevice` to specify the camera to use when the `source` is [ImageSource.camera].
   /// The `preferredCameraDevice` is ignored when `source` is [ImageSource.gallery]. It is also ignored if the chosen camera is not supported on the device.
-  /// Defaults to [CameraDevice.rear].
+  /// Defaults to [CameraDevice.rear]. 
+  ///
```
Comment (cyanglaz) [inline packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart:136]: Actually, I think you were right. It was related to the comment above. Whatever you did was correct. My mistake :) Can we revert this change? Sorry for the confusion.
Same above.

**Review by cyanglaz — APPROVED** [review]
LGTM! CI seems unhappy with formatting.

File: packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart (hunk 4 above)
Comment (mvanbeusekom) [reply packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart]: No problem, will revert back

**Comment by valconetti** [comment]
> All (the pull request submitter and all commit authors) CLAs are signed, **but** one or more commits were authored or co-authored by someone other than the pull request submitter.
> 
> We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that by leaving a comment that contains only `@googlebot I consent.` in this pull request.
> 
> _Note to project maintainer:_ There may be cases where the author cannot leave a comment, or the comment is not properly detected as consent. In those cases, you can manually confirm consent of the commit author(s), and set the `cla` label to `yes` (if enabled on your project).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fflutter%2Fplugins%2Fpull%2F3037) for more info**.

@googlebot I consent.

**Comment by valconetti** [comment]
@googlebot I consent.

**Comment by mvanbeusekom** [comment]
@googlebot I consent.

**Review by cyanglaz — APPROVED** [review]
LGTM!
